lib/nosql_client_impl.js

```diff
@@ -436,9 +476,9 @@ array in the output for listRoles operation', null, req);
         });
     }
 
-    /* for testing */
-    _getSerialVersion() {
-        return this._client.getSerialVersion();
+    //used for testing
```

Comment:
Did you add a testing mechanism for forcing the protocol version to a specific version (e.g. V3)?

Reply:
I haven't added it here, is it required?  Note that the unit tests have --kv, --proxy and --proto parameters to test against specific protocol version (there is a mapping between kv and proxy to protocol version).  This way the test run specifies exactly what we expect.  However, to test older protocol version, they would have to run with older server version.  I didn't think we need to test older protocol with new server, is this needed?  I didn't do that for C# driver either.